cloudferrylib/os/compute/nova_compute.py

```diff
@@ -383,6 +383,16 @@ def convert_instance(instance, cfg, cloud):
         else:
             server_group = None
 
+        config_drive = instance.config_drive
```

Comment:
But it's intended to be used for configuration, not API. Do you have any example where OpenStack API service use oslo_config.types.Boolean to parse something from API?